src/appleseed/renderer/modeling/environmentedf/environmentedf.cpp

```diff
@@ -99,7 +99,17 @@ bool EnvironmentEDF::on_frame_begin(
                 Matrix4d::make_rotation(rotation)));
     }
 
+    if (m_params.get_optional<bool>("env_cast_shadows", true))
+        m_env_cast_shadows = true;
+    else
+        m_env_cast_shadows = false;
+
     return true;
 }
 
+bool EnvironmentEDF::get_cast_shadows() const
+{
+    return m_env_cast_shadows;
```

Comment:
`m_cast_shadows`

Reply:
The variable defined is m_env_cast shadows. Do I need to rename it?